src/components/utils/test/thread_test.cc

```diff
@@ -54,11 +54,15 @@ const std::string kThreadName("THREAD");
 
 class ThreadTestDelegate : public threads::ThreadDelegate {
  public:
-  ThreadTestDelegate() : thread_stopped_(false), thread_started_(false) {}
+  ThreadTestDelegate()
+      : thread_stopped_(false), thread_started_(false), starts_counter_(0) {}
```

Comment:
@orhan-mehmedov There are two simple, but commonly used variables in this test class - thread_stopped_ and thread_started_ . In order to avoid confusion, I think it's better to stick to similar names, highlighting that one is used when thread has started and another one - when stopped.